Tests/SwiftDocCTests/Model/SemaToRenderNodeTests.swift

```diff
@@ -1922,6 +1922,22 @@ Document
             XCTAssertEqual(renderNode.metadata.platforms?.first?.isBeta, false)
         }
         
+        // Symbol with an empty set of availbility items.
+        
+        do {
+            
+            let (bundle, context, _) = try makeTestBundle(currentPlatforms: [
+                "Custom Name": PlatformVersion(VersionTriplet(100, 0, 0), beta: true)
+            ])
+            let reference = ResolvedTopicReference(bundleIdentifier: bundle.identifier, path: "/documentation/MyKit/MyClass", sourceLanguage: .swift)
```

Comment:
FYI: The test helper above is returning the same reference, so you could get it from there instead:
```suggestion
            
            let (bundle, context, reference) = try makeTestBundle(currentPlatforms: [
                "Custom Name": PlatformVersion(VersionTriplet(100, 0, 0), beta: true)
            ])
```